packages/bundler-plugin-core/src/sentry/sourcemaps.ts

```diff
@@ -1,38 +1,66 @@
 import path from "path";
-import fs from "fs";
+import fs, { Stats } from "fs";
+import glob from "glob";
+import { InternalIncludeEntry } from "../options-mapping";
 
 export type FileRecord = {
   name: string;
   content: string;
 };
 
-export function getFiles(path: string, allowedExtensions: string[]): FileRecord[] {
-  const includedFiles = getAllIncludedFileNames(path, allowedExtensions, []);
+export function getFiles(fpath: string, includeEntry: InternalIncludeEntry): FileRecord[] {
+  let fileStat: Stats;
+  const p = path.isAbsolute(fpath) ? fpath : path.resolve(process.cwd(), fpath);
```

Comment:
l: Let's try to give this a more expressive name for self-doc reasons

Suggestion (but feel free to call it something else):
```suggestion
  const absolutePath = path.isAbsolute(fpath) ? fpath : path.resolve(process.cwd(), fpath);
```

Reply:
Yeah that was pure laziness during development. Wanted to change that variable before pushing the PR - thanks for pointing out!